### PR TITLE
React native version update to 0.68.2

### PR DESCRIPTION
**Pull request by dan-arvi**
# Related Issue

Update all package dependency to the latest RN major version (RN version 0.68.2)

# Description

Updated the react native version to 0.68.2 and all the package dependencies.

### CODE REVIEW

File: source/community/reactnative/package.json [hunk 1]
```diff
@@ -53,8 +54,9 @@
     "adaptivecards-templating": "1.0.0-rc.0",
     "antlr4ts": "^0.5.0-alpha.3",
     "assert": "^2.0.0",
-    "react-native-video": "^4.4.2",
-    "react-native-keyboard-aware-scroll-view": "^0.9.5"
+    "patch-package": "^6.4.7",
```
Comment (Zaidos9) [inline source/community/reactnative/package.json]: should `patch-package` be under dependencies or devDependencies?

File: source/community/reactnative/package.json [hunk 2]
```diff
@@ -53,8 +54,9 @@
     "adaptivecards-templating": "1.0.0-rc.0",
```
Comment (Zaidos9) [inline source/community/reactnative/package.json:55]: Please upgrade dateTimePicker and Picker as well.
```
"@react-native-community/datetimepicker": "^6.0.2",
"@react-native-picker/picker": "2.4.1"
```

**Review by Zaidos9 — CHANGES_REQUESTED** [review]

**Review by pragadeeshk — APPROVED** [review]
Looks good to me

File: source/community/reactnative/package.json (hunk 2 above)
Comment (pragadeeshk) [reply source/community/reactnative/package.json:55]: @dan-arvi resolved this

File: source/community/reactnative/package.json (hunk 1 above)
Comment (pragadeeshk) [reply source/community/reactnative/package.json]: Updated

**Review by Zaidos9 — APPROVED** [review]
LGTM

**Pull request merged**